sovrin_node/server/node.py

```diff
@@ -565,7 +569,12 @@ def storeTxnAndSendToClient(self, reply):
          client requests it.
         """
         result = reply.result
+        if result[TXN_TYPE] in (SCHEMA, ISSUER_KEY):
+            result = deepcopy(result)
```

Comment:
@rajeshkalaria80 deepcopy can be refactored. I will use result as it is. But I need `if` condition because I need to serialize data with txn type `schema` or `issuer_key`.